src/extensions/default/InlineBorderRadiusEditor/css/main.less

```diff
@@ -0,0 +1,412 @@
+@sansFontFamily: "Open Sans" , Helvetica, Arial, sans-serif;
```

Comment:
We need to confirm with @flukeout the font choice here so it matches what we use elsewhere.